core/arch/arm/plat-zynqmp/platform_config.h

```diff
@@ -39,6 +39,9 @@
 #error "Pager not supported for zynqmp"
 #endif
 
+#define DRAM0_BASE		0
+#define DRAM1_BASE		0x800000000
```

Comment:
Is this valid if `CFG_DDR_SIZE` is <= 0x80000000?

Reply:
DRAM1_BASE memory map should not be configured when DRAM size is not over 2GiB.

DRAM0_BASE address gives view for first 2 GiB and then rest is in DRAM1_BASE. 

Note that there are quite many 0 digits in DRAM1_BASE ;).

Reply:
OK, please drop the DRAM1_BASE when it's not valid.

Reply:
While resolving 32 vs 64 bit compiling issue decided it is a good idea to hide 64 bit addresses when compiling for 32 bits so added guard for that.